src/api/models/index.js

```diff
@@ -17,7 +17,9 @@ const models = {
   Highlight: sequelize.import('./highlight'),
   Following: sequelize.import('./following'),
   Comment: sequelize.import('./comment'),
-  Bookmark: sequelize.import('./bookmark')
+  Bookmark: sequelize.import('./bookmark'),
+  LikeDislike: sequelize.import('./likesDislike'),
```

Comment:
Why not use ``Reaction`` instead of ``LikeDislike``

Reply:
Because we are specifically liking and disliking but the reaction is not specific at all

Reply:
@murediane I would go with commentReaction, but this can work too, I suppose

Reply:
On second thought, @murediane I think we should change the model and table name to `commentReaction`